src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -778,6 +778,9 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
         mapped_table = (fi_addr_t *) av_attr.map_addr;
         for (i = 0; i < size; i++) {
             MPIDI_OFI_AV(&MPIDIU_get_av(0, i)).dest = mapped_table[i];
+            if (MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS) {
```

Comment:
This should also check for runtime capability sets per our previous conversation I think.